packages/connect/src/create-store.js

```diff
@@ -21,12 +21,18 @@ const getSnapshot = obj => {
 
 const convertToAction = (fn, instance) => (...args) => {
   const first = fn(instance);
-  if (first instanceof Promise)
-    return new Promise(resolve => first.then(() => resolve()));
+  if (first instanceof Promise) {
+    return new Promise((resolve, reject) =>
+      first.then(resolve).catch(err => reject(err))
```

Comment:
The test is fine. Just don't return anything in the `resolve` here. Actions do not return anything to force people to use state and avoid an antipattern.